code/game/objects/items/melee/baton.dm

```diff
@@ -519,9 +519,19 @@
 
 /obj/item/melee/baton/security/attack_self(mob/user)
 	if(cell?.charge >= cell_hit_cost)
-		active = !active
-		balloon_alert(user, "turned [active ? "on" : "off"]")
-		playsound(src, SFX_SPARKS, 75, TRUE, -1)
+		if(SSsecurity_level.get_current_level_as_number() >= SEC_LEVEL_RED)
```

Comment:
We have a signal for security level changes, you can tap into that to automatically shut batons off when it drops from red.

However, what's stopping crew from roundstart keeping it on red all day?

Reply:
RP

Reply:
you expect the admin team to enforce that? they barely stop AIs from cracking the upload open for the HOS when nuke ops are announced, RP is not a priority for our admins to enforce